core/comment.js

```diff
@@ -178,71 +208,99 @@ Blockly.Comment.prototype.resizeBubble_ = function() {
  */
 Blockly.Comment.prototype.setVisible = function(visible) {
   if (visible == this.isVisible()) {
-    // No change.
     return;
   }
   Blockly.Events.fire(
       new Blockly.Events.Ui(this.block_, 'commentOpen', !visible, visible));
-  if ((!this.block_.isEditable() && !this.textarea_) ||
-      Blockly.utils.userAgent.IE) {
-    // Steal the code from warnings to make an uneditable text bubble.
-    // MSIE does not support foreignobject; textareas are impossible.
```

Comment:
I would keep the comment about IE not supporting foreign objects in the same place we check for the useragent. ie: in this new code it's createBubble_